bot/cogs/moderation/scheduler.py

```diff
@@ -84,7 +84,8 @@ def mod_log(self) -> ModLog:
         """Apply an infraction to the user, log the infraction, and optionally notify the user."""
         infr_type = infraction["type"]
         icon = utils.INFRACTION_ICONS[infr_type][0]
-        reason = infraction["reason"]
+        # Truncate reason when it's too long to avoid raising error on sending ModLog entry
+        reason = textwrap.shorten(infraction["reason"], width=1900, placeholder="...")
```

Comment:
Isn't 1900 cutting it a bit too close? That's only a bit over 100 characters for everything else.

Reply:
Why is being done here? The truncation should be happening in the methods that are sending the messages, e.g. `utils.notify_infraction` and `mod_log.send_log_message`. This way you can use the actual character limit instead of having to guess at how long it needs to be in order to not raise an error.

Reply:
@sco1 I applied this to `utils.notify_infraction`, but I can't do this in `mod_log.send_log_message`, because this take argument `text`, where is the full content of embed description (that include reason).

Reply:
Yes, that’s the point. You’re focusing on truncating only the reason, arbitrarily guessing at the length you need to truncate to, when you can just truncate the entire string that these methods generate before sending.

Reply:
@sco1 But like when reason is too long, this will cut `Expiry` (`apply_infraction`) from ModLog entry, because this is after reason. To make this, I have to put everything else first and after this reason.

Reply:
Yes.